supervised/loglizer/benchmarks/HDFS_bechmark.py

```
@@ -0,0 +1,75 @@
+    (x_tr, y_train), (x_te, y_test) = dataloader.load_HDFS(struct_log,
+                                                           window='session', 
+                                                           train_ratio=0.5,
+                                                           split_type='uniform')
+    benchmark_results = []
+    for _model in run_models:
+        print('Evaluating {} on HDFS:'.format(_model))
+        if _model == 'PCA':
+            feature_extractor = preprocessing.FeatureExtractor()
+            x_train = feature_extractor.fit_transform(x_tr, term_weighting='tf-idf', 
+                                                      normalization='zero-mean')
+            model = PCA()
```

Ruff (0.8.2)

`PCA` may be undefined, or defined from star imports

(F405)